pontoon/base/static/js/translate.js

```diff
@@ -2666,7 +2686,6 @@ var Pontoon = (function (my) {
           entity: entity.pk,
           translation: serializedTranslation,
           plural_form: submittedPluralForm,
-          original: entity['original' + self.isPluralized()],
           ignore_check: $('#quality').is(':visible') || !syncLocalStorage || entity.format === 'ftl',
```

Comment:
This reminds me that we should (possibly as a followup) remove the "homemade" checks we use for FTL files.